pkg/packagekit/autoupdates.jsx

```diff
@@ -249,67 +248,128 @@ function getBackend(forceReinit) {
  * Properties:
  * onInitialized(enabled): (optional): callback once backend knowsn whether automatic updates are enabled
  */
-export default class AutoUpdates extends React.Component {
+export const AutoUpdatesBody = (props) => {
+    const { enabled, type, day, time } = props;
+    const days = {
+        "": "everyday",
+        mon: "Monday",
+        tue: "Tuesday",
+        wed: "Wednesday",
+        thu: "Thursday",
+        fri: "Friday",
+        sat: "Saturday",
+        sun: "Sunday"
+    };
+
+    let str;
+    if (enabled) {
+        str = type == "security" ? _("Security updates ") : _("Updates ");
+        str += day == "" ? cockpit.format(_("will be applied $0 at $1"), days[day], time) : cockpit.format(_("will be applied every $0 at $1"), days[day], time);
```

Comment:
- `days` are not localized.
- space at the end of `Security updates ` and `Updates ` are super likely to get broken by translators
- But more importantly translating `will be applied $0 at $1` is difficult to translate without any context. `every Monday` in slovak is `každý pondelok` while `every Wednesday` is `každú stredu`. Also translating just `Wednesday` (as in the list above) I wouls translate `streda` and not `stredu` so the resulting string would be `budú aplikované každý streda` which sounds like you speak Slovak for the first time.

Either moment offers something for this or we need to do this in better way. One option is to list all the options for all 8 days and twice for `every` or without. 16 strings, not ideal... Other option would be to rethink the format of that string and write it in such way that it would work with neutral forms.

Reply:
I cannot think of a format of that string in a neutral form. Perhaps we could use that object and add every in front of it? Like:     
```
const days = {
        "": _("everyday"),
        mon: _("every Monday"),
        tue: _("every Tuesday"),
        wed: _("every Wednesday"),
        thu: _("every Thursday"),
        fri: _("every Friday"),
        sat: _("every Saturday"),
        sun: _("every Sunday"),
    };
```